src/main/java/core/org/akaza/openclinica/core/LiquibaseConfig.java

```diff
@@ -11,6 +11,7 @@
  * Created by yogi on 3/22/17.
  */
 @Configuration
+@ComponentScan("ScheduledJobConfig.class")
```

Comment:
why do we need this?

Reply:
The dependsOn annotation doesn't work when it's on its own on a component, you have to set componentScan along with it.